src/Functions/Public/iaas/New-vRAMachineAttachedDisk.ps1

```diff
@@ -185,9 +185,9 @@
                         $machineResponse = Invoke-vRARestMethod -URI "$APIUrl`?`$filter=name eq '$Name'`&`$select=id" -Method GET
                         $machineId = $machineResponse.content[0].id
 
-                        $Response = Invoke-vRARestMethod -URI "$APIUrl`/$machineId`/disks" -Method POST -Body $Body
+                        $RestResponse = Invoke-vRARestMethod -URI "$APIUrl`/$machineId`/disks" -Method POST -Body $Body
 
-                        CalculateOutput
+                        CalculateOutput $blockDeviceId $CompletionTimeout $WaitForCompletion $RestResponse
```

Comment:
Variable should be PascalCase

`$blockDeviceId` ->` $BlockDeviceId`

Reply:
fixed with a mass find/replace in next commit.